src/main/scala/cromwell/engine/db/slick/ExecutionComponent.scala

```diff
@@ -99,6 +107,12 @@ trait ExecutionComponent {
       if !(execution.status === ExecutionStatus.NotStarted.toString || execution.status === ExecutionStatus.Done.toString)
     } yield execution)
 
```

Comment:
No FK to the reference execution from which results were taken, but per our in person discussion that should be okay for the forseeable future.